torch/jit/__init__.py

```diff
@@ -48,6 +48,8 @@ def _parse_env(name, default, true_message, false_message):
 _jit_script_compile = torch._C._jit_script_compile
 BatchTensor = torch._C._jit.BatchTensor
 compiled_weak_fns = weakref.WeakKeyDictionary()
+weak_script_methods = weakref.WeakKeyDictionary()
+weak_modules = weakref.WeakKeyDictionary()
```

Comment:
Document what this holds.  The code in _try_get_weak_module is confusing because there is no documentation for what weak_modules holds.